modular_skyrat/modules/organs/lungs.dm

```diff
@@ -1,49 +1,49 @@
 /obj/item/organ/lungs/cold
 	name = "Cold Adapted Lungs"
 	icon = 'modular_skyrat/modules/organs/lungs.dmi'
-	desc = "A set of lungs adapted to low temperatures, though they are more susceptible to high temperatures"
+	desc = "A set of lungs adapted to low temperatures, though they are more susceptible to high temperatures, breathing cold gases heals you very lightly depending how cold they are."
 	icon_state = "lungs_cold"
-	cold_message = "There is a slightly painful, though bearable, cold sensation with every breath you take"
-	cold_level_1_threshold = 208
-	cold_level_2_threshold = 200
-	cold_level_3_threshold = 170
-	cold_level_1_damage = COLD_GAS_DAMAGE_LEVEL_1 //Keep in mind with gas damage levels, you can set these to be negative, if you want someone to heal, instead.
-	cold_level_2_damage = COLD_GAS_DAMAGE_LEVEL_1
-	cold_level_3_damage = COLD_GAS_DAMAGE_LEVEL_2
-	cold_damage_type = BURN
+	cold_message = "There is a slightly painful, though bearable, smoothing and good cold sensation with every breath you take"
```

Comment:
```suggestion
	cold_message = "There is a slightly painful, though bearable, smoothing and good cold sensation with every breath you take."
```
Punctuation good.
I'd also probably change this message, because it feels a little strange.

Reply:
i toke the punctuation on purpose.
If you put it its says on chat
"There is a slightly painful, though bearable, smoothing and good cold sensation with every breath you take.on your (lungs name here)"

Reply:
so it was like 
"i toke the punctuation on purpose.
If you put it its says on chat
"There is a slightly painful, though bearable, smoothing and good cold sensation with every breath you take!On your (lungs name here)"